daffodil-test/src/test/scala/org/apache/daffodil/usertests/RC-Test.scala

```diff
@@ -17,30 +17,23 @@
 
 package org.apache.daffodil.usertests
 
-import org.junit.Test
+// import org.junit.Test
 import org.apache.daffodil.tdml.Runner
 import org.junit.AfterClass
 
-object RCTest5 {
+object RCTest {
   val testDir = "/org/apache/daffodil/usertests/"
-  val runner = Runner(testDir, "test-5.tdml.xml")
   val runner6 = Runner(testDir, "test-6.tdml.xml")
 
   @AfterClass def shutDown(): Unit = {
-    runner.reset
     runner6.reset
   }
 
 }
 
-class RCTest5 {
+class RCTest {
 
-  import RCTest5._
-
-  @Test def test5p(): Unit = { runner.runOneTest("parse-test-5") }
-
-  // DAFFODIL-2217
```

Comment:
Does this also resolve part of DAFFODIL-2217, or were these issues actually related to 2219 are were labeled incorrectly?

Reply:
See below.